templates/pulpcore-docker-apache.conf.erb

```diff
@@ -2,5 +2,8 @@
   SSLRequire %{SSL_CLIENT_S_DN_CN} eq "admin" 
```

Comment:
Is this still needed? Doesn't the API handle authentication now? Should we pass the the variable to the API backend?

Reply:
added!